site/pages/docs/ref/add-cal/add-cal-en.hbs

```diff
@@ -25,6 +25,12 @@
 	<li><a href="../../../demos/add-cal/add-cal-fr.html">French example</a></li>
 </ul>
 
+<h2>Accessibility testing</h2>
```

Comment:
Accessibility testing are an "Evaluation and report", please move those link under that heading bellow.

Reply:
Commenting to resolve conversation, changes were applied in previous commit